swagger_spec_validator/validator20.py

```diff
@@ -35,10 +36,14 @@ def validate_spec(spec_json, _spec_url=None):
     """
     validate_json(spec_json, 'schemas/v2.0/schema.json')
 
+    # Dereference all $refs so we don't have to deal with them
+    fix_malformed_model_refs(spec_json)
```

Comment:
I haven't quantified the latency but if there no side-effects in bravado because of it (may be a manual example run should suffice), then i am ok with it.